packages/cli/package.json

```diff
@@ -83,6 +85,9 @@
       },
       "ui:create": {
         "description": "create a user interface powered by different front end frameworks and Coveo Headless"
+      },
+      "search": {
+        "description": "perform queries against a coveo index"
```

Comment:
How do you think in the future the topic subdivision with org etc?

One could argue that an index is something relative to an org, and therefore that it should be `org:search:dump`

Reply:
Fair enough !